cmd/nerdctl/volume.go

```diff
@@ -56,9 +56,5 @@ func getVolumeStore(cmd *cobra.Command) (volumestore.VolumeStore, error) {
 	if err != nil {
 		return nil, err
 	}
-	dataStore, err := clientutil.DataStore(dataRoot, address)
-	if err != nil {
-		return nil, err
-	}
-	return volumestore.New(dataStore, ns)
+	return volume.Store(ns, dataRoot, address)
```

Comment:
seems this `getVolumeStore` is no longer needed as `volume.Store` should be preferred.